src/main/resources/org/jenkinsci/test/acceptance/docker/fixtures/SshdContainer/Dockerfile

```diff
@@ -7,11 +7,11 @@ FROM ubuntu:artful-20171019
 # install SSHD
 RUN apt-get update -y && \
     apt-get install -y \
-        openssh-server=1:7.5p1-10
+        openssh-server=1:7.5p1-10ubuntu0.1
```

Comment:
Seems unrelated?

Reply:
That is the reason the container does no longer build so I am fixing things as I find them broken.

Reply:
This part is covered in #14.